src/s2/base/commandlineflags_declare.h

```diff
@@ -0,0 +1,46 @@
+// Copyright Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef S2_BASE_COMMANDLINEFLAGS_DECLARE_H_
+#define S2_BASE_COMMANDLINEFLAGS_DECLARE_H_
+
+#include <string>
+
+#include "s2/base/integral_types.h"
+
+#ifdef S2_USE_GFLAGS
+
+#include <gflags/gflags.h>
+
+#else  // !defined(S2_USE_GFLAGS)
+
+#define DECLARE_bool(name) \
```

Comment:
Re https://github.com/google/s2geometry/pull/297/files/be8a56d2952660b3c977a0a70ba861e597dbe9e0#r1065262491

Looks like this should be `S2_DECLARE_` and we need it for the gflags branch, too.

Reply:
Oh, really

Reply:
Fixed.